src/components/Profile/ProfileComponent.jsx

```diff
@@ -116,13 +124,29 @@ const ProfileComponent = ({ contactProfile }) => {
             setInputValue={setNickname}
             edit={edit}
           />
+          <ProfileInputField inputName="WebId" inputValue={webId} />
         </Box>
         {!contactProfile && (
-          <ProfileEditButtonGroup
-            edit={edit}
-            handleCancelEdit={handleCancelEdit}
-            handleEditInput={handleEditInput}
-          />
+          <Box sx={{ display: 'flex', flexDirection: 'row', gap: '10px' }}>
+            <ProfileEditButtonGroup
+              edit={edit}
+              handleCancelEdit={handleCancelEdit}
+              handleEditInput={handleEditInput}
+            />
+            <Typography sx={{ marginTop: '8px' }}>
+              <a href={signupLink} rel="noopener noreferrer" target="_blank">
+                Your Invite Link
+              </a>
+              <IconButton aria-label="Copy Invite Link" edge="end">
+                <ContentCopyIcon
+                  onClick={() => {
```

Comment:
I'm getting this error:

<img width="208" alt="2024-02-19 (1)" src="https://github.com/codeforpdx/PASS/assets/83156697/908df4fd-906a-48cb-94fc-5c99ba5c4ad5">

It resolved by simply moving 

```
onClick={() => {
  navigator.clipboard.writeText(signupLink);
  addNotification('success', 'Invite link copied to clipboard');
}}
```

to the `ContentCopyIcon`. It still retains functionality.

Reply:
Fixed